CHANGELOG-3.7.md

```diff
@@ -8,87 +8,118 @@ We recommend that you use the release notes as a guide for migrating your
+  - [C# Changes](#c-changes-10)
   - [JavaScript Changes](#javascript-changes-3)
   - [MATLAB Changes](#matlab-changes-1)
-  - [Python Changes](#python-changes-3)
+  - [Python Changes](#python-changes-4)
   - [Ruby Changes](#ruby-changes-2)
   - [Swift Changes](#swift-changes-3)
 - [Changes in Ice 3.7.3](#changes-in-ice-373)
   - [General Changes](#general-changes-3)
-  - [C++ Changes](#c-changes-9)
-  - [C# Changes](#c-changes-10)
-  - [Java Changes](#java-changes-3)
+  - [C++ Changes](#c-changes-11)
+  - [C# Changes](#c-changes-12)
+  - [Java Changes](#java-changes-4)
   - [JavaScript Changes](#javascript-changes-4)
   - [MATLAB Changes](#matlab-changes-2)
-  - [Python Changes](#python-changes-4)
+  - [Python Changes](#python-changes-5)
 - [Changes in Ice 3.7.2](#changes-in-ice-372)
   - [General Changes](#general-changes-4)
-  - [C++ Changes](#c-changes-11)
-  - [C# Changes](#c-changes-12)
-  - [Java Changes](#java-changes-4)
+  - [C++ Changes](#c-changes-13)
+  - [C# Changes](#c-changes-14)
+  - [Java Changes](#java-changes-5)
   - [JavaScript Changes](#javascript-changes-5)
   - [MATLAB Changes](#matlab-changes-3)
   - [Objective-C Changes](#objective-c-changes)
   - [PHP Changes](#php-changes-3)
-  - [Python Changes](#python-changes-5)
+  - [Python Changes](#python-changes-6)
 - [Changes in Ice 3.7.1](#changes-in-ice-371)
   - [General Changes](#general-changes-5)
-  - [C++ Changes](#c-changes-13)
-  - [C# Changes](#c-changes-14)
-  - [Java Changes](#java-changes-5)
+  - [C++ Changes](#c-changes-15)
+  - [C# Changes](#c-changes-16)
+  - [Java Changes](#java-changes-6)
   - [JavaScript Changes](#javascript-changes-6)
   - [MATLAB Changes](#matlab-changes-4)
   - [Objective-C Changes](#objective-c-changes-1)
   - [PHP Changes](#php-changes-4)
-  - [Python Changes](#python-changes-6)
+  - [Python Changes](#python-changes-7)
   - [Ruby Changes](#ruby-changes-3)
 - [Changes in Ice 3.7.0](#changes-in-ice-370)
   - [General Changes](#general-changes-6)
-  - [C++ Changes](#c-changes-15)
-  - [C# Changes](#c-changes-16)
-  - [Java Changes](#java-changes-6)
+  - [C++ Changes](#c-changes-17)
+  - [C# Changes](#c-changes-18)
+  - [Java Changes](#java-changes-7)
   - [JavaScript Changes](#javascript-changes-7)
   - [Objective-C Changes](#objective-c-changes-2)
   - [PHP Changes](#php-changes-5)
-  - [Python Changes](#python-changes-7)
+  - [Python Changes](#python-changes-8)
   - [Ruby Changes](#ruby-changes-4)
 
+# Changes in Ice 3.7.10
+
+These are the major changes since Ice 3.7.9.
+
+## C++ Changes
+
+- Fixed failures building from source with G++ 13. (https://github.com/zeroc-ice/ice/issues/1489)
+
+- Fixed the C++11 library names produced by Windows builds. (https://github.com/zeroc-ice/ice/issues/1502)
+
+- Removed UWP support. (https://github.com/zeroc-ice/ice/issues/1503)
+
+## C# Changes
+
+- Fixed generated code for compatability with .NET 8. (https://github.com/zeroc-ice/ice/issues/1548)
+
+## Java Changes
+
+- Add support for JDK 21.
```

Comment:
For consistency with the rest of the file:
```suggestion
- Added support for JDK 21.
```
We always say "added support for X".